en/flight_controller/cuav_v5_nano.md

```diff
@@ -102,6 +102,46 @@ The three power rails are: **POWER1**and **USB**.
 Any multicopter / airplane / rover or boat that can be controlled with normal RC servos or Futaba S-Bus servos.
 The complete set of supported configurations can be seen in the [Airframes Reference](../airframes/airframe_reference.md).
 
+
+## Known Issues
+
+#### GPS not compatible with other devices (Critical) - Batch 01
+
+The *Neo v2.0 GPS* recommended for use with *CUAV V5+* and *CUAV V5 nano* is not fully compatible with other Pixhawk flight controllers (specifically, the buzzer part is not compatible and there may be issues with the safety switch).
+The GPS will not work with other flight controllers, and is the only GPS unit that can be used with the *CUAV V5+* and *CUAV V5 nano*.
+<!-- Nano/80/V5_NANO_RC01_2019_05_29 -->
+
+- *Found:* Batch 01
+- *Fixed:* -
+
+#### PM2 cannot power the flight controller
+
+`PM2` can only measure battery voltage and current, but **not** power the flight controller.
+
+> **Warning** We recommend you do not connect a power module to this connector when using PX4.
+
+- *Found:* Batch 01
+- *Fixed:* -
+
+#### Volt regulation varies greater than +/- 5%
+
+The 5 volt pins on all connectors will be lower when powered from USB than the Power Module (the pins will measure approximately 4.69V when only powered by USB, and 5.28 Volts when connected to the Power Module).
+
+> **Note** We recommend that when using USB with the *v5 Nano* you *also connect the power module* (to avoid under-powering any connected peripherals.
```

Comment:
@davids5 FYI, I assume that "(ADD PROPS DISCLAIMER)" means "make this text a proper and visible disclaimer". If not, can you expand on what you wanted here?

Reply:
Perhaps I used the wrong word. We are telling them to test with a PM on the bench so we should also Add a warning to Remove Props before bench testing.

Reply:
Perhaps I used the wrong word. We are telling them to test with a PM on the bench so we should also Add a warning to Remove Props before bench testing.

Reply:
Thanks. Not so much a matter of "wrong word" as "wrong context for me" - since we're talking about boards I was guessing maybe PROPS was an acronym for some board port I couldn't find information on :-)

I'll fix.